scheduler/ecs/ecs.go

```diff
@@ -308,6 +316,7 @@ type ecsProcessManager struct {
 	cluster     string
 	serviceRole string
 	ecs         *ecsutil.Client
+	config      Config
```

Comment:
The `Config` struct should really only be used by the NewScheduler factory. Can we change this to:

```go
logDriver string
// or
logConfiguration *ecs.logConfiguration
```

And then change the signature of `taskDefinitionInput` to `func taskDefinitionInput(p *scheduler.Process, logConfiguration *ecs.LogConfiguration)`